Dockerfile

```
@@ -16,6 +16,10 @@ COPY 30-remote-site-setup.sh /etc/osg/image-config.d/
 
 COPY 99-container.conf /usr/share/condor-ce/config.d/
 
+# TODO: Drop this after implementing non-root Gratia probes
+# https://opensciencegrid.atlassian.net/browse/SOFTWARE-3975
+RUN chmod 777 /var/lib/gratia/tmp
```

```suggestion
RUN chmod 1777 /var/lib/gratia/tmp
```

I know it's in a container, but it seems to me not a great idea to make a 777 dir anywhere without the sticky bit.